--- qemu-2.12.0/target/i386/translate.c	2018-06-12 21:38:39.934900998 +0800
+++ qemu-2.12.0/target/i386/translate.c	2018-06-12 15:48:04.516413622 +0800
@@ -5027,6 +5027,11 @@
             break;
         case 2: /* call Ev */
             /* XXX: optimize if memory (no 'and' is necessary) */
+#ifdef DEBUG_DISAS
+            if (qemu_loglevel_mask(CPU_LOG_TB_IN_ASM) && qemu_log_in_addr_range(s->base.pc_first)) {
+                qemu_log("[+] indir\n");
+            }
+#endif
             if (dflag == MO_16) {
                 tcg_gen_ext16u_tl(cpu_T0, cpu_T0);
             }
